src/vimmy2.js

```diff
@@ -197,6 +202,12 @@
       if ( key === 'shift-h' ) goBackHistory();
       if ( key === 'shift-l' ) goForwardHistory();
 
+      if ( !isNaN(key) ) {
+        MULTIPLIER = MULTIPLIER * 10 + Number(key);
```

Comment:
My recommendation is to skip the `Number` call here, and instead to jump to line 283, and remove the `.toString` call there, since it’s pretty pointless.